### PR TITLE
Add voice assistant file

### PR DESCRIPTION
**Pull request by sofiiadidovych**

### CODE REVIEW

File: javascript/javascript1/week4/index.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
```
Comment (ajgcph2019) [inline javascript/javascript1/week4/index.html:1]: Very good. Nice to see you have used string methods like 'startsWith',endsWith and included extra commands

File: javascript/javascript1/week4/voice_assistant.js [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+            if (map[i].country === country) {
+               return `The capital of ${map[i].country} is ${map[i].capital}`;
+            }
+        }
+        return `I don't know what is the capital of ${country}`;
+    } else if (command.startsWith('what is')) {
+        const startIndex = 'What is '.length;
+        const lastIndex = command.length;
+        const evaluation = command.slice(startIndex, lastIndex);
+        return eval(evaluation);
```
Comment (ajgcph2019) [inline javascript/javascript1/week4/voice_assistant.js:68]: eval is not recommended.
Check [https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/eval](url)
You can also change command to array of words then get numbers and operation like this for example 

 const commandWords = command.split(" ");
    const number1 = parseInt(commandWords[2]);
    const number2 = parseInt(commandWords[4]);
    const operation = commandWords[3];
    if (number1 && number2) {
      switch (operation) {
        case "+":
          return number1 + number2;
        case "-":
          return number1 - number2;
        case "*":
          return number1 * number2;

**Review by ajgcph2019 — CHANGES_REQUESTED** [review]

File: javascript/javascript1/week4/index.html (hunk 1 above)
Comment (sofiiadidovych) [reply javascript/javascript1/week4/index.html:1]: thank you)

File: javascript/javascript1/week4/voice_assistant.js (hunk 2 above)
Comment (sofiiadidovych) [reply javascript/javascript1/week4/voice_assistant.js]: I changed the file. Thank you for your feedback :)

**Pull request merged**